WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -515,6 +516,28 @@ public void setHelpContext(String faqId, String faqSection) {
     @Override
     public void saveCredentialsInSmartLock(@Nullable final String username, @Nullable final String password,
                                            @NonNull final String displayName, @Nullable final Uri profilePicture) {
+        if (getLoginMode() == LoginMode.SELFHOSTED_ONLY) {
```

Comment:
minor nit: can we add a `// TODO` here so we make sure to mark this as something that needs be removed?

Reply:
Hmm, I'd consider that an enhancement (opened https://github.com/wordpress-mobile/WordPress-Android/issues/7300 to track that) since the selfhosted flow is not supposed to be a primary _WPCOM_ flow (and SmartLock-for-Passwords is only utilized for WPCOM anyway). Adding a `// TODO` would indicate that this is something that _needs_ to be done, but I'm not sure that's the case. WDYT?

Reply:
Ah - I understand the intention of this change better now. You're right, that'd be an enhancement. Thanks for the clarification!